comparisons/validate.C

```diff
@@ -984,59 +984,59 @@ void validateEvents(TString step, TString file, TString refFile, TString r="RECO
       //CTPPS
       tbr="TotemFEDInfos_totemRPRawToDigi_RP_";
       plotvar(tbr+recoS+".obj@.size()");
-      plotvar(tbr+recoS+".obj.getFEDId()");
-      plotvar(tbr+recoS+".obj.getOptoRxId()");
-      plotvar(tbr+recoS+".obj.getFSize()");
+      plotvar(tbr+recoS+".obj.fedId()");
```

Comment:
@forthommel  thanks for looking at this as well. Did you close the PR intentionally?

This code will have to support running diffs on the old releases as well as the new ones.
The approach to satisfy this is to *add* the new methods.